server.js

```diff
@@ -33,8 +71,61 @@ app.post('/api/createUser', (req, res) => {
 	});
 
 	connection.end();
+
+})
+
+app.get('/api/getEvents', (req, res) => {
+
+	let connection = mysql.createConnection(config);
+	//will prob need to edit sql statement to account for search features later
+	//may need to modify later so that won't load them all as an array 
+	//need to make more efficient 
+	let sql = `select firstName, lastName, events.name, events.description, events.location, events.date
+	from shchowdh.users, shchowdh.events
+	where events.creatorID = users.id
+	and date >= curdate()
+	order by date`; 
+	console.log(sql);
+
+	connection.query(sql, (error, results, fields) => {
+		if (error) {
+			return console.error(error.message);
+		}
+
+		let string = JSON.stringify(results);
+		//let obj = JSON.parse(string);
+		res.send({ express: string });
+
+	});
+	connection.end();
 });
 
+app.post('/api/userEmailSearch', (req, res) => {
```

Comment:
This should be a `GET` request, would you be able to update this line to: `app.get`. This is my mistake 😅

Reply:
Can it be a GET request if it has a body? I think I got an error when I tried to switch it due to the email part

Reply:
You're right, `GET` requests can't have a body. This should be a `POST` request since its a filtered search rather than a general search.